### ISSUE TITLE
Hyperlink to external website from book sidebar

### ISSUE THREAD

**Issue by dvirlar2** [quarto-dev/quarto-cli#1973]
Hi there, Quarto team! 

When working in a Quarto book, is it possible right now to have a "chapter" hyperlink to a different website? Specifically, I'm thinking about how Bookdown uses specific code in the [bookdown::gitbook:](https://github.com/dgrtwo/tidy-text-mining/blob/master/_output.yml) section of the `_output.yml` file (code linked) to create the "View book source" link on the [Tidy Text Mining](tidytextmining.com) site. 

I'm trying to convert my team's training manual from a bookdown to a quarto book, and I've been racking my brain trying to figure this out to link to our extra reference manual from the main training document. I feel like I keep just barely skirting around the solution, but I just can't seem to put things together 😅  I know I can get creative and just make the document a website rather than a book and get things to work that way, but I'm curious if there's a current way to make this work from a book, specifically. 

Apologies if this already been asked, I've perused all the discussions and issues, but still could have missed it! Thanks!

**Comment by jjallaire** [comment]
No, I don't think this is possible right now. The best you could do would be to create a chapter that is empty save for a JavaScript redirect:

````html
```{=html}
<script type="text/javascript">
window.location.replace("https://example.com");
</script>
```
````

You can also insert headers and footers for the right margin sidebar: https://quarto.org/docs/websites/website-tools.html#headers-footers

Note that there is also a feature for GitHub links: https://quarto.org/docs/websites/website-navigation.html#github-links

**Comment by mcanouil** [comment]
While, it's not possible to add "chapters" or "parts" as external links, the "other-link" features works in the TOC sidebar (right).

- https://quarto.org/docs/output-formats/html-basics.html#code-links-and-other-links
